client/src/components/Cart-icon/cart-icon.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
```

Comment:
context api being leveraged to handle the state of `togglehidden` when user clicks the cart icon. The cart icon also displays the number of items are being added to the cart by the user in the forms of an integer